{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": []
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "cve@mitre.org",
      "ID": "CVE-2019-4547"
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "IBM Security Directory Server 6.4.0 generates an error message that includes sensitive information about its environment, users, or associated data. IBM X-Force ID: 165949."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": []
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "ibm-sds-cve20194547-info-disc (165949)",
          "refsource": "XF",
          "tags": [],
          "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/165949"
        },
        {
          "name": "https://www.ibm.com/support/pages/node/6356607",
          "refsource": "CONFIRM",
          "tags": [],
          "url": "https://www.ibm.com/support/pages/node/6356607"
        }
      ]
    }
  },
  "impact": {},
  "lastModifiedDate": "2020-10-29T16:27Z",
  "publishedDate": "2020-10-29T16:15Z"
}